packages/rich-text-editor/src/RichTextEditor/RichTextEditor.tsx

```diff
@@ -54,11 +65,34 @@ export const RichTextEditor: React.VFC<RichTextEditorProps> = props => {
     <>
       {labelText && <Label id={labelId} labelText={labelText} />}
       {/* TODO: add a bit of margin here once we have a classNameOverride on Label */}
-      <div
-        ref={editorRef}
-        className={classnames(styles.editor, classNameOverride)}
-        {...restProps}
-      />
+      <div className={styles.editorWrapper}>
+        {controls && (
+          <Toolbar aria-controls="toolbar-ref-id" aria-label="Text formatting">
```

Comment:
This `toolbar-ref-id` isn't referencing anything yet I think? We'll want to make sure it's unique as well in the case of multiple RTEs on one screen, probably with `uuid`.